src/helpers/translation.ts

```diff
@@ -1622,6 +1635,14 @@ export const codeMessages = defineMessages({
     id: 'code.E_PAYMENT_GATEWAY',
     defaultMessage: 'E_PAYMENT_GATEWAY',
   },
+  E_SETUP_PAYPAL: {
+    id: 'code.E_SETUP_PAYPAL',
+    defaultMessage: 'paypal環境未設定',
+  },
```

Comment:
NITS: `E_PAYPAL_SETUP` is better

Reply:
i see. we can discuss about this in the code review meeting.
i prefer E_NAMESPACE_ACTION pattern